### DISCUSSION TITLE
List of changed icon names in v2

### DISCUSSION THREAD

**Discussion by MichaelBurgess** [tailwindlabs/heroicons#760] [Help] [answered]
Hi,

The release notes for v2 say: `A number of icons have also been renamed. For example, the "mail" icon has been renamed to "envelope" and the "search" icon has been renamed to "magnifying glass".`

Is there is a list of all the icons that have changed name with their new name please? I want to assess the impact of changing from v1 to v2 in our codebase.

Thanks.

**Comment by albofish** [comment, marked as answer]
https://github.com/tailwindlabs/heroicons/releases/tag/v2.0.0 and search for `Update icon names`

**Reply by MichaelBurgess** [reply]
Thanks! Missed that first time around.

**Reply by whitespacecode** [reply]
Why change the naming of so much icons? Just use the same name as heroicons.dev so it's easier to search them
